app/views/users/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Index of Users</h1>
+<% @users.each do |user| %>
+  <br>
+  <%= user %>
+  <br>
+  <%= user.email %>
+  <br>
+<% end %>
+<%= link_to("Articles", articles_path) %>
```

Comment:
Doesn't look like you have a route for `users#index`. Is this file actually being used anywhere?